tests/security/atsec/dbus_fuzzer.pm

```diff
@@ -0,0 +1,113 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Run 'DBus fuzzer' test case of ATSec test suite
+# Maintainer: xiaojing.liu <xiaojing.liu@suse.com>
+# Tags: poo#109978
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use atsec_test;
+use Mojo::Util 'trim';
+use Data::Dumper;
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    # Install the required packages
+    zypper_call('in glib2-devel libffi-devel gcc make');
+
+    # Compile
+    assert_script_run("cd $atsec_test::code_dir/pentest/dfuzzer-master/src");
+    assert_script_run('make');
+
+    my $output = script_output('./dfuzzer -l 2>&1');
+
+    # Parse the output and push the items to related array
+    my @bus_list;
+    foreach my $line (split(/\n/, $output)) {
+        $line = trim($line);
+        if ($line =~ /SESSION\s+BUS|SYSTEM\s+BUS|Session bus not found/) {
+            next;
+        }
+        elsif ($line =~ /Exit status:\s+(\d)/) {
+            if ($1 != 0) {
+                record_info('Fail to get system bus information', $output, result => 'fail');
+                $self->result('fail');
+            }
+        } else {
+            push(@bus_list, $line);
+        }
+    }
+
+    record_info('Result of dfuzzer -l', Dumper(\@bus_list));
+
+    # Analyse the results
+    my %hash_white_list = map { $_ => 1 } @atsec_test::white_list_for_dbus;
+    my @unknown_bus_name = grep { !$hash_white_list{$_} } (@bus_list);
+
+    # After filtering there should be no unknown name.
+    if (scalar(@unknown_bus_name) > 0) {
+        record_info('There are unknow bus name', Dumper(\@unknown_bus_name), result => 'fail');
+        $self->result('fail');
+    }
+
+    # Create a directory to restore the log message generated by dfuzzer
+    my $log_dir = 'logs';
```

Comment:
Minor comments, use more complex dir name

Reply:
This directory is under the test directory, I think simple name is ok.